crates/chain-config/src/config/state.rs

```diff
@@ -364,14 +408,56 @@ impl AddTable<ContractsLatestUtxo> for StateConfigBuilder {
 }
 
 impl AsTable<Transactions> for StateConfig {
-    fn as_table(&self) -> Vec<TableEntry<Transactions>> {
-        Vec::new() // Do not include these for now
+    fn as_table(&self, chain_id: &ChainId) -> Vec<TableEntry<Transactions>> {
+        self.transactions
+            .iter()
+            .map(|tx| TableEntry {
+                key: tx.id(chain_id),
+                value: tx.clone().into(),
+            })
+            .collect()
```

Comment:
It seems if we remove transactions from the `StateConfig` we don't need to pass `ChainId`=)

Reply:
True.

Reply:
3a721e5bf